changelog/fix_error_plugin_no_bundler.md

```diff
@@ -0,0 +1 @@
+* [#13867](https://github.com/rubocop/rubocop/issues/13867): Fix an error for plugins when not running RuboCop through Bundler. ([@earlopain][])
```

Comment:
```suggestion
* [#13867](https://github.com/rubocop/rubocop/issues/13867): Fix an error for plugins when not running RuboCop through Bundler. ([@Earlopain][])
```

Reply:
I've used the current capitalization in all previous entries.